server/src/chainanalysis-service.ts

```diff
@@ -8,17 +9,49 @@ export interface UTXO {
 
 export class ChainAnalysisService {
 
-    async getTransactionDetailsFromBlockStream(transactionId: string) : Promise<any> {
```

Comment:
We are getting UTXOs not transaction details

Reply:
Yeah, used the dummy method pending when the getUtxoDetails function will be ready.